tests/unit/tools/test_ts_checker_base.py

```
@@ -0,0 +1,337 @@
+    Returns:
+        The ToolResult produced by ``check``.
+    """
+    with patch(
+        "lintro.plugins.execution_preparation.verify_tool_version",
+        return_value=None,
+    ):
+        with patch.object(
+            plugin,
+            "_run_subprocess",
+            return_value=subprocess_return,
+        ):
```

Ruff (0.15.20)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)